<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EntryForm</class>
 <widget class="QMainWindow" name="EntryForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EntryForm</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollBar" name="horizontalScrollBar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>710</y>
      <width>1011</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="verticalScrollBar">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>0</y>
      <width>20</width>
      <height>711</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QWidget" name="drawingWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1011</width>
      <height>711</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>25</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_map"/>
    <addaction name="actionOpen_map"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionSave"/>
    <addaction name="actionProperties"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuUnits">
    <property name="title">
     <string>Modules</string>
    </property>
    <addaction name="actionUnit_Module"/>
    <addaction name="actionTile_Module"/>
    <addaction name="actionObject_Modules"/>
    <addaction name="actionProjectile_Modules"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuUnits"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_map">
   <property name="text">
    <string>New map</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionOpen_map">
   <property name="text">
    <string>Open map</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as..</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionProperties">
   <property name="text">
    <string>Properties</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionUnit_Module">
   <property name="text">
    <string>Unit Modules</string>
   </property>
  </action>
  <action name="actionTile_Module">
   <property name="text">
    <string>Tile Modules</string>
   </property>
  </action>
  <action name="actionObject_Modules">
   <property name="text">
    <string>Object Modules</string>
   </property>
  </action>
  <action name="actionProjectile_Modules">
   <property name="text">
    <string>Projectile Modules</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
